tests/nrf5340_audio/contin_array/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+#
```

Comment:
`Unit tests and on-target tests for nRF5340 Audio application`

There should be commit per test, please split accordingly.

Reply:
We were told to split this commit in five, by folder. This has been approved by @gopiotr who is codeowner for tests/